keyboards/program_yoink/keymaps/split_bar/keymap.c

```diff
@@ -0,0 +1,70 @@
+/* Copyright 2020 melonbred
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _LAYER1,
+    _LAYER2
+};
+
+// Defines Combos for use in the keymap
+enum combo_events {
+  COMBO_SLSH,
+};
```

Comment:
There is partial combo config for this keymap, but `key_combos` is missing. It should either be removed, or enabled.

Reply:
Updated keymap to complete combo config. 
Applied changes from local machine.